### PR TITLE
fix(@desktop/wallet): Added missing LoadingGraphComponent in qmldir

### PR DESCRIPTION
**Pull request by Khushboo-dev-cpp**
### What does the PR do

Moved the LoadingGraphComponent to correct location

### Affected areas

Desktop App

### StatusQ checklist

- [ ] add documentation if necessary (new component, new feature)
- [ ] update sandbox app
  - in case of new component, add new component page
  - in case of new features, add variation to existing component page
  - nice to have: add it to the demo application as well
- [ ] test changes in both light and dark theme?

### CODE REVIEW

**Review by 0x-r4bbit — APPROVED** [review]

File: ui/imports/shared/views/qmldir [hunk 1]
```diff
@@ -10,3 +10,4 @@ PasswordView 1.0 PasswordView.qml
 ProfileDialogView 1.0 ProfileDialogView.qml
 AssetsView 1.0 AssetsView.qml
 HistoryView 1.0 HistoryView.qml
+LoadingGraphView 1.0 LoadingGraphView.qml
```
[the remark this answers is not in the document]
Comment (Khushboo-dev-cpp) [reply ui/imports/shared/views/qmldir]: good point! done

**Review by stefandunca — APPROVED** [review]

**Review by caybro — COMMENTED** [review]
This doesn't look like a missing qmldir entry but rather a file move

**Comment by caybro** [comment]
Meaning I'm missing the actual qmldir change, even if the file got moved

**Comment by caybro** [comment]
The sanity checker fails with:
```
[2023-01-31T09:03:46.167Z] gmake[4]: *** No rule to make target '../src/StatusQ/Components/StatusChatListCategory.qml', needed by 'sanity_checker/qrc_statusq.cpp'.  Stop.
```

**Comment by Khushboo-dev-cpp** [comment]
> Meaning I'm missing the actual qmldir change, even if the file got moved

@caybro wasn't needed after I moved the file as it is in the same folder as where it is used. will update description

**Comment by Khushboo-dev-cpp** [comment]
> The sanity checker fails with:
> 
> ```
> [2023-01-31T09:03:46.167Z] gmake[4]: *** No rule to make target '../src/StatusQ/Components/StatusChatListCategory.qml', needed by 'sanity_checker/qrc_statusq.cpp'.  Stop.
> ```

@caybro do you know why :D ? I was just wondering how this is linked in any way

**Comment by caybro** [comment]
So the failure got introduced with https://github.com/status-im/status-desktop/pull/9187 :/ Pls fix it here, eventhough it's unrelated; otherwise it will always be failing the "imports" test

**Comment by Khushboo-dev-cpp** [comment]
> So the failure got introduced with #9187 :/ Pls fix it here, eventhough it's unrelated; otherwise it will always be failing the "imports" test

make sense doing it

**Comment by Khushboo-dev-cpp** [comment]
another error form other PR in imports test


```
[2023-01-31T09:20:40.584Z] (qrc:/StatusQ/Components/StatusChatListAndCategories.qml:70:13: Type StatusChatList unavailable

[2023-01-31T09:20:40.584Z]                 StatusChatList {

[2023-01-31T09:20:40.584Z]                 ^, qrc:/StatusQ/Components/StatusChatList.qml:11:1: module "utils" is not installed

[2023-01-31T09:20:40.584Z]     import utils 1.0 as LocalUtils

[2023-01-31T09:20:40.584Z]     ^)

[2023-01-31T09:20:40.584Z] (qrc:/StatusQ/Components/StatusChatList.qml:11:1: module "utils" is not installed

[2023-01-31T09:20:40.584Z]     import utils 1.0 as LocalUtils

[2023-01-31T09:20:40.584Z]     ^)

[2023-01-31T09:20:40.584Z] make: *** [Makefile:663: run-statusq-sanity-checker] Error 1
```

**Comment by Khushboo-dev-cpp** [comment]
> another error form other PR in imports test
> 
> ```
> [2023-01-31T09:20:40.584Z] (qrc:/StatusQ/Components/StatusChatListAndCategories.qml:70:13: Type StatusChatList unavailable
> 
> [2023-01-31T09:20:40.584Z]                 StatusChatList {
> 
> [2023-01-31T09:20:40.584Z]                 ^, qrc:/StatusQ/Components/StatusChatList.qml:11:1: module "utils" is not installed
> 
> [2023-01-31T09:20:40.584Z]     import utils 1.0 as LocalUtils
> 
> [2023-01-31T09:20:40.584Z]     ^)
> 
> [2023-01-31T09:20:40.584Z] (qrc:/StatusQ/Components/StatusChatList.qml:11:1: module "utils" is not installed
> 
> [2023-01-31T09:20:40.584Z]     import utils 1.0 as LocalUtils
> 
> [2023-01-31T09:20:40.584Z]     ^)
> 
> [2023-01-31T09:20:40.584Z] make: *** [Makefile:663: run-statusq-sanity-checker] Error 1
> ```

Ive simple added a readonly proeprty holding the same value that was expected from Constants.qml file. This could also be an enum though.

**Review by alaibe — APPROVED** [review]

**Pull request merged**